services/cognito/spec/factories/metabase_cards.rb

```diff
@@ -0,0 +1,6 @@
+FactoryBot.define do
```

Comment:
rubocop will complain here

Reply:
@rpbaltazar pushed requested changes. ready for review.